layouts/index.html

```diff
@@ -0,0 +1,157 @@
+{{ define "body" }}
+<div class="container-fluid">
+    {{ $recentEvents := (.Site.GetPage "/events").Pages | first 4 }}
+    <div class="row justify-content-lg-start justify-content-center pt-3 mb-lg-5 mb-3">
```

Comment:
If we're going to center the `<hr />`s, can we also center the header?
```suggestion
    <div class="row justify-content-center pt-3 mb-lg-5 mb-3">
 ```

Reply:
I feel like the title should be kept left aligned.

Reply:
I see - I think it just feels off to me because the spacing is uneven around the title + header text, but not enough so that it feels like it was an intentional choice.

![image](https://user-images.githubusercontent.com/45278276/187468391-63c36a84-f89a-4bb5-a30f-ab1904926424.png)